src/firebuild/CMakeLists.txt

```diff
@@ -31,6 +31,9 @@ if (SANITIZE)
   endif()
 endif()
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -I${tsl-hopscotch-map_INCLUDE_DIRS}")
```

Comment:
Somehow `include_directories("${tsl-hopscotch-map_INCLUDE_DIRS}")` did not work. :-(